### DISCUSSION TITLE
My student developer package was accepted but it's been 4 days and I still haven't received it!

### DISCUSSION THREAD

**Discussion by LiMaKen** [community/community#129398] [GitHub Education] [answered]
### Hi everyone,

Hey everyone,

I signed up for this package and received it on the 18th, on the 22nd I still haven't received the student developer package.

Given this situation, should I wait a bit longer, or would it be advisable to take further action? I would appreciate any guidance or recommendations on the best course of action at this time.

Thank you in advance!
![image](https://github.com/community/community/assets/86464924/b035dce9-6544-4343-89c7-2db79eceb587)

**Comment by mzsprojects** [comment]
Hi! Even I haven't received the benefits yet. Its been two days.
It took a **week** for [this](https://github.com/orgs/community/discussions/129270) person.
I think we may have to wait the same.

**Comment by LiMaKen** [comment]
Hello!
Maybe Github has too many orders, we should wait

**Comment by davevad93** [comment, marked as answer]
Hi @LiMaKen , 

It may take up to a couple of weeks for the benefits to become available, the status will change to: 

"`Congrats! Your request for @LiMaKen was approved and your benefits are now available`"

It usually takes only a few days but the waiting time may vary depending on different factors such as the volume of applications.
